apps/frontend/src/libs/components/select/select.tsx

```diff
@@ -5,43 +5,76 @@ import {
 	type Path,
 	type PathValue,
 } from "react-hook-form";
-import ReactSelect from "react-select";
+import ReactSelect, { type ActionMeta, type OnChangeValue } from "react-select";
 
 import { getValidClassNames } from "~/libs/helpers/helpers.js";
-import { useFormController } from "~/libs/hooks/hooks.js";
+import { useCallback, useFormController } from "~/libs/hooks/hooks.js";
 import { type SelectOption } from "~/libs/types/types.js";
 
 import styles from "./styles.module.css";
 
 type Properties<TFieldValues extends FieldValues, TOptionValue> = {
 	control: Control<TFieldValues, null>;
+	isLabelVissible?: boolean;
```

Comment:
I would say it's better to call it isLabelHidden, because visible is preferred option
```suggestion
	isLabelHidden?: boolean;
```